SU-SWS/decanter-react
# ADAPT-3250: Adjustments to Tailwind configuration approach to support JIT Mode.

# READY FOR REVIEW

# Summary
- Updates Tailwind inclusion approach. Rather than import fully-compiled CSS from decanter, it now pulls in individual Tailwind plugins and components, as we are doing in the saa_alumni repo.
- Also updates Tailwind to use Just-in-time (JIT) compiler.

# Needed By (Date)
- By end of sprint

# Urgency
- Not aware of any particular urgency. But there may be other tickets waiting on this change as a dependency. 

# Steps to Test

1. Just view Storybook in the preview environment, and compare to https://decanter-react.netlify.app/ to ensure configuration changes did not introduce style regressions.

# Affected Projects or Products
- Yes - any projects using decanter-react will be eventually be impacted by this change after it is merged and projects update to latest version. 

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ADAPT-3250

# Other Notes
- Some npm packages had to be updated to due to dependency requirements for tailwindcss.
- I decided to stay on the previous version of decanter for now. (1.0.0-alpha.7) Let me know if I should upgrade to 1.0.0-beta.0, which is the version of decanter used on saa_alumni.
